src/scripts/core/models/background.ts

```diff
@@ -67,66 +83,54 @@ export class Background extends Container {
 
     public enterShip(): void {
         this._ship.alpha = 1;
-        gsap.from(this._ship, {
-            x: this._viewWidth + this._ship.width,
-            duration: 3
-        })
-    }
-
-    public setWaveHeightLow(): void {
-        this.backgroundSpriteTweenManager(-0.2);
-    }
-
-    public setWaveHeightMedium(): void {
-        this.backgroundSpriteTweenManager(-0.15);
-    }
-
-    public setWaveHeightHigh(): void {
-        this.backgroundSpriteTweenManager(0);
+        // gsap.from(this._ship, {
+        //     x: this._viewWidth + this._ship.width,
+        //     duration: 3
+        // })
     }
 
     private initWave(sprite: Sprite, width: number, height: number): void {
         sprite.width = width;
         sprite.height = height;
     }
 
-    private backgroundSpriteTweenManager(waveHeightMultiplier: number): void {
-        const farTime = Math.floor(Math.random() * 5) + 3;
-        const midTime = Math.floor(Math.random() * 5) + 3;
-        const closeTime = Math.floor(Math.random() * 5) + 3;
+    public setWaveHeight(farWaveHeightMod: number, midWaveHeightMod: number, closeWaveHeightMod: number, duration: number): void {
+        this.resetSplashTimeline(farWaveHeightMod, midWaveHeightMod ,closeWaveHeightMod, duration);
+    }
 
-        if (this._splash) { this._splash.kill(); }
-        this._splash = gsap.timeline();
```

Comment:
As we discussed briefly the removal of this looks like your tweens are not being cleared between resets, meaning you're stacking multiple tweens with each reset.